CMakeLists.txt

```diff
@@ -32,6 +32,30 @@ if (NOT CMAKE_CONFIGURATION_TYPES)
     set(CMAKE_CONFIGURATION_TYPES "Debug;Release;RelWithDebInfo;MinSizeRel" CACHE STRING "Configs")
 endif()
 
+# Determine UNIX distribution
+if(NOT WIN32)
+    file(READ "/etc/os-release" ETC_ISSUE)
+    string(REGEX MATCH "Debian|Ubuntu|CentOS|RHEL|SLES" DIST ${ETC_ISSUE})
+    set(USE_TBB Off)
+
+    if (DIST STREQUAL "Debian")
+        message(STATUS "Debian OS detected!")
+        add_compile_definitions(DEBIAN_DISTRO)
+        set(USE_TBB On)
+    elseif(DIST STREQUAL "Ubuntu")
+        message(STATUS "Ubuntu OS detected!")
+        add_compile_definitions(DEBIAN_DISTRO)
+        set(USE_TBB On)
+    elseif(DIST STREQUAL "CentOS")
+        message(STATUS "CENTOS detected!")
+    elseif(DIST STREQUAL "Red Hat")
+        message(STATUS "Red Hat detected!")
+    elseif(DIST STREQUAL "SUSE")
+        message(STATUS "SLES detected!")
+    endif()
+endif()
```

Comment:
None of this is needed, We already detect a correct tbb version and use it if its available: https://github.com/ROCm/AMDMIGraphX/blob/develop/src/CMakeLists.txt#L291

Reply:
Removed